src/main/java/com/Acrobot/Breeze/Utils/InventoryUtil.java

```diff
@@ -1,15 +1,11 @@
 package com.Acrobot.Breeze.Utils;
 
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-
 import com.Acrobot.ChestShop.Configuration.Properties;
 import org.bukkit.inventory.Inventory;
 import org.bukkit.inventory.ItemStack;
 
+import java.util.*;
+
```

Comment:
This should be reverted seeing as the file isn't touched otherwise now.

Reply:
This is still not reverted.

Reply:
Ah, I reverted it but my IDE will optimise imports on any modified files when committing. Will fix on next push.